sdk/identity/identity/README.md

```diff
@@ -1,8 +1,18 @@
 ## Azure Identity client library for JavaScript
 
-This library simplifies authentication against Azure Active Directory for Azure SDK libraries.
-It provides a set of `TokenCredential` implementations which can be passed into SDK libraries
-to authenticate API requests. It supports token authentication using an Azure Active Directory [service principal](https://docs.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli) or [managed identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview).
+The Azure Identity library provides Azure Active Directory token authentication support across the Azure SDK. It provides a set of [TokenCredential](https://docs.microsoft.com/javascript/api/@azure/core-auth/tokencredential) implementations which can be used to construct Azure SDK clients which support AAD token authentication.
+
+This library currently provides credentials for:
+
+- [Service principal authentication](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals)
+- [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview)
+- [Device code authentication](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-device-code)
+- Interactive browser authentication, based on [OAuth2 authentication code](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow)
```

Comment:
The current InteractiveBrowserCredential uses implicit grant flow: https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-implicit-grant-flow . The new one will allow users to opt in to the Implicit Grant Flow an will default into this Auth Code flow you references here 🌞

Reply:
Isnt that change in behavior only for when InteractiveBrowserCredential is used in the browser rather than in node? What would be a good one line description with an optional link be good here?

Reply:
We need some time to define that part. I think we can say something like `based on the authentication flows: Auth Code and Implicit Grant` with links in each. Just a thought. I can update this with more info later this week.

Reply:
Would it be ok for the scope of this PR to use the link you have on implicit grant flow and you can change it when we merge the new one?

```suggestion
- Interactive browser authentication, based on [OAuth2 authentication code](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-implicit-grant-flow)
```

Reply:
Or, I can also go with simply `Interactive browser authentication, based on OAuth2 authentication code.` and have no links

Reply:
Going with https://github.com/Azure/azure-sdk-for-js/pull/13416/commits/f9eb9a471857f6b058d9b5099c0ee9d164d97a0e